lib/jekyll-katex/configuration.rb

```diff
@@ -10,7 +10,7 @@ class Configuration
       LOG_TOPIC = 'Katex Configuration:'
       CONFIG_DEFAULTS = {
         js_filename: 'katex.min.js',
-        js_path: File.join(Jekyll::Katex::LIB_ROOT, 'assets/js/'),
+        js_path: File.join(Jekyll::Katex::LIB_ROOT, 'assets/js'),
```

Comment:
```suggestion
        js_path: File.join(Jekyll::Katex::LIB_ROOT, 'assets', 'js'),
```

I guess part of the issue is hard-coding the path separator. Would prefer removing its use completely and simply rely on `File.join`.

Reply:
Similar change might be preferred on line 26:

```ruby
katex_js = Dir.glob("#{File.join(js_path, '**', js_filename}").first
```

Reply:
Technically, `File.join` is smart in the sense that it doesn't simply join strings but produces valid paths. For example,
```ruby
File.join('root', 'assets/', '/js')
# => "root/assets/js"
```

Reply:
Yep! I still think it's best to allow `File.join` to determine the separators and avoid hard-coding separators since we probably should support Windows users as well.

The issue with the bad path is probably more due to the lack of `File.join` on L26.

Reply:
FYI: Ruby on Windows uses `/` as separator as well.
And yes, `File.join` in L26 would've automatically resolved this as well.

Reply:
Ah good to know. Mind fixing it in both places? I think it makes the code more robust and readable.

Please also provide any manual verification possible that this issue is fixed.

Appreciate the quick fix!

Reply:
This repo needs a proper test-suite.. :wink:
I'm not a first-hand user of this plugin. Came across this while debugging some other issue for a Jekyll user. So, I won't be able to provide a manual verification at the moment.

Reply:
@ashmaroli Definitely needs a lot of work. Testing and automated build and publications have been on the TODO forever. Hopefully, I get to it when I return in May.

Reply:
Good Luck! :smiley: